framework/src/meshgenerators/RenameBoundaryGenerator.C

```diff
@@ -24,33 +24,12 @@ RenameBoundaryGenerator::validParams()
   InputParameters params = MeshGenerator::validParams();
 
   params.addRequiredParam<MeshGeneratorName>("input", "The mesh we want to modify");
-  params.addDeprecatedParam<std::vector<BoundaryID>>(
-      "old_boundary_id",
-      "Elements with these boundary ID(s) will be given the new boundary ID(s) and/or name(s). You "
-      "must supply either 'old_boundary_id' or 'old_boundary_name'.",
-      "Use 'old_boundary' instead of 'old_boundary_id'.");
-  params.addDeprecatedParam<std::vector<BoundaryName>>(
-      "old_boundary_name",
-      "Elements with these boundary name(s) will be given the new boundary ID(s) and/or name(s). "
-      "You must supply either 'old_boundary_id' or 'old_boundary_name'.",
-      "Use 'old_boundary' instead of 'old_boundary_name'");
-  params.addDeprecatedParam<std::vector<BoundaryID>>(
-      "new_boundary_id",
-      "The new boundary ID(s) for the elements defined by "
-      "'old_boundary_id' or 'old_boundary_name'.",
-      "Use 'new_boundary' instead of 'new_boundary_id'.");
-  params.addDeprecatedParam<std::vector<BoundaryName>>(
-      "new_boundary_name",
-      "The new boundary name(s) for the elements defined by "
-      "'old_boundary_id' or 'old_boundary_name'. If 'new_boundary_id' is not provided and a "
-      "boundary with the given name does not exist, a new boundary ID will be created.",
-      "Use 'new_boundary' instead of 'new_boundary_name'.");
-
-  params.addParam<std::vector<BoundaryName>>(
```

Comment:
These options while deprecated are very useful. 

I have a mesh which uses them, and we should check the VTBin case there are others

Reply:
VTB is good, see draft PR below.

Reply:
yeah my bad. It was the RenameBlockGenerator that I use the deprecated options for

Reply:
but good idea to check